tests/data_context/test_data_context_v013.py

```diff
@@ -480,6 +484,11 @@ def test_in_memory_data_context_configuration(
             ],
         }
     }
+
+    # Roundtrip through schema validation to remove any illegal fields add/or restore any missing fields.
+    project_config_dict = dataContextConfigSchema.dump(project_config_dict)
+    project_config_dict = dataContextConfigSchema.load(project_config_dict)
```

Comment:
@cdkini Agreed -- the fact that adding it was necessary to make things correct (since the updated architecture is now tighter and more formal) indicates that this should be refactored (agreed -- for a separate PR at a later date).  Thanks.

Reply:
Agreed. This shouldn't go in this PR but something we should keep an eye out for! Thanks 🙇🏽